Covid19Radar/Covid19Radar/ViewModels/Settings/DebugPageViewModel.cs

```diff
@@ -0,0 +1,160 @@
+﻿/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+using System;
+using Acr.UserDialogs;
+using Covid19Radar.Services;
+using Prism.Navigation;
+using Xamarin.Forms;
+
+namespace Covid19Radar.ViewModels
+{
+    public class DebugPageViewModel : ViewModelBase
+    {
+        private readonly IUserDataService userDataService;
+        private readonly ITermsUpdateService termsUpdateService;
+        private readonly IExposureNotificationService exposureNotificationService;
+
+        private string _debugInfo;
+        public string DebugInfo
+        {
+            get { return _debugInfo; }
+            set { SetProperty(ref _debugInfo, value); }
+        }
+        public async void Info(string ex = "")
+        {
+            string os;
+            switch (Device.RuntimePlatform)
+            {
+                case Device.Android:
+                    os = "Android";
+                    break;
+                case Device.iOS:
+                    os = "iOS";
+                    break;
+                default:
+                    os = "unknown";
+                    break;
+            }
```

Comment:
`Device.RuntimePlatform`をそのまま使えば、OSプラットフォームの文字列をとることができます。Windowsとかも入ってしまいますが、そもそもビルドできないので考慮しなくて良いと思います。

```suggestion
            string os = Device.RuntimePlatform;
```